templates/zerver/emails/confirm_registration.html

```diff
@@ -1,36 +1,32 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
-<html lang="en">
-<head>
-        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-        <title>Zulip</title>
-</head>
-<body>
-<table width="80%" style="align:center; max-width:800px" align="center">
-    <tr>
-        <td style="font-size:16px; font-family:Helvetica;">
-            <p>Hello there.
-            </p>
-            <p>
-                You recently signed up for Zulip. Awesome!
-            </p>
-            <p>
-                To complete signup, visit this link below:
-                <br />
-                <a href="{{ activate_url }}" style="color:#08c">{{ activate_url }}</a>
-            </p>
-            <p>
-            {% if verbose_support_offers %}
-                Feel free to give us a shout at <a href="mailto:{{ support_email }}" style="color:#08c">{{ support_email }}</a>, if you have any questions.
-            {% else %}
-                If you are having issues, please contact your Zulip administrator at <a href="mailto:{{ support_email }}" style="color:#08c">{{ support_email }}</a>.
-            {% endif %}
-            </p>
-            <p>
-                Cheers,<br />
-                The Zulip Team
-            </p>
-        </td>
-    </tr>
-</table>
-</body>
-</html>
+{% extends "zerver/emails/email_base.html" %}
+
```

Comment:
As a high-level comment, we don't want to add too many extra newlines, since it reduces the density of the code in the editor. In the case of emails, we also print these emails to the console, so fewer empty newlines are better.

I would write this part as 
```
{% extends "zerver/emails/email_base.html" %}

{% block header %} <h3>almost there!</h3> {% endblock %}
{% block content %}
<p>Hello there.
```

similar comment for the other template files.